src/js/msp.js

```diff
@@ -147,25 +234,33 @@ var MSP = {
     clearListeners: function() {
         this.listeners = [];  
     },
-    send_message: function (code, data, callback_sent, callback_msp, callback_onerror) {
-        if (code === undefined) {
-            debugger;
+    crc8_dvb_s2: function(crc, ch) {
+        crc ^= ch;
+        for (let ii = 0; ii < 8; ii++) {
+            if (crc & 0x80) {
+                crc = ((crc << 1) & 0xFF) ^ 0xD5;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assignment to function parameter 'crc'.](https://app.codacy.com/gh/emuflight/EmuConfigurator/pullRequest?prid=5635143)